ios/app/MBXAppDelegate.m

```diff
@@ -1,11 +1,27 @@
 #import "MBXAppDelegate.h"
 #import "MBXViewController.h"
+#import <mbgl/ios/MapboxGL.h>
 #import <mbgl/ios/MGLMapboxEvents.h>
 
 @implementation MBXAppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
+    // Set Access Token
+    NSString *accessToken = [[NSProcessInfo processInfo] environment][@"MAPBOX_ACCESS_TOKEN"];
+    if (accessToken) {
+        // Store to preferences so that we can launch the app later on without having to specify
+        // token.
+        [[NSUserDefaults standardUserDefaults] setObject:accessToken forKey:@"access_token"];
```

Comment:
Nit: A common strategy for user defaults keys is to make them match the names of the constants that define them. So you’d define a constant called `MGLMapboxAccessTokenDefaultsKey` with the value `MGLMapboxAccessToken` in MGLTypes.h and use that constant here. Obviously it doesn’t matter as long as you’re consistent, but it’s one less thing to worry about naming.